drivers/slipdev/slipdev.c

```diff
@@ -67,11 +68,90 @@ static inline void slipdev_unlock(void)
     }
 }
 
-static void _slip_rx_cb(void *arg, uint8_t byte)
+static inline void _slipdev_stdio_add_to_frame(slipdev_t * dev, uint8_t byte) {
+    if (IS_USED(MODULE_SLIPDEV_STDIO)) {
+        if (dev->config.uart == slipdev_params[0].uart) {
+            isrpipe_write_one(&stdin_isrpipe, byte);
+        }
+    }
+}
+
+static inline bool _slipdev_config_start_frame(slipdev_t * dev) {
+#ifdef MODULE_SLIPDEV_CONFIG
+    /* try to create new configuration / CoAP frame */
+    return crb_start_chunk(&dev->rb_config);
+#else
+    (void) dev;
+    return 1;
+#endif
+}
+
+static inline void _slipdev_config_end_frame(slipdev_t * dev) {
+#ifdef MODULE_SLIPDEV_CONFIG
+    crb_end_chunk(&dev->rb_config, true);
+    thread_flags_set(thread_get(dev->coap_server_pid), 1);
```

Comment:
I would suggest adding a define for the thread flags.
But since you only use a single flag, couldn't the `thread_flags_wait_any(1)` be replace by a simple `mutex_lock()`?